src/lib/auth.js

```diff
@@ -4,23 +4,42 @@ import { z } from "zod";
 import { env } from "@/lib/env";
 
 export const { auth, handlers } = NextAuth({
+  secret: process.env.AUTH_SECRET,
+  session: {
+    strategy: "jwt",
+    maxAge: 3600 // 1 hour
```

Comment:
@IvanKoshan Please, check if 1 hour maxAge matches with business requirements.

Reply:
@yataras I believe it does not matter for now
because auth will be used only for the admin panel.
and it is just the first version of admin panel.

Reply:
@BohdanLeshchyshyn OK